app/views/assignments/peer_review.html.erb

```diff
@@ -0,0 +1,324 @@
+<%= render partial: 'peer_submissions_table', formats: [:'js.jsx'], handlers: [:erb] %>
+<script>
+  var invite_modal = null;
+
+  function invite() {
+    invite_modal.open();
+    document.getElementById('invite_member').value = '';
+    document.getElementById('invite_member').focus();
+  }
+
+  jQuery(document).ready(function() {
+    invite_modal = new ModalMarkus('#invite_dialog');
+  });
+</script>
+
+<div class='title_bar'>
+  <h1>
+    <%= t('student.peer_assignment',
+          short_identifier: @assignment.short_identifier) %>
+  </h1>
+</div>
+
+<div class='student_columns'>
+  <div id='student_column_left'>
+    <div class='block'>
+      <%= render partial: 'read' %>
+    </div>
+
+    <div class='block'>
+      <h2><%= t('submissions') %></h2>
+
+      <div class='block_content'>
+        <% if Assignment.find_by_id(@assignment.parent_assignment_id).past_all_collection_dates? %>
```

Comment:
Oh, this is also probably not the best check. Instead, why don't you check to see if the student's grouping actually has any peer reviews assigned to it?